frontend/src/app/rating/rating.module.ts

```diff
@@ -0,0 +1,21 @@
+import { CommonModule } from '@angular/common';
```

Comment:
Can you add comments for all the files? You can check my previous PR for commenting procedure.

Reply:
i will add some comments :)